tests/integration/test_keeper_ipv4_fallback/test.py

```diff
@@ -0,0 +1,38 @@
+import os
+import pytest
+import subprocess
+from helpers.cluster import ClickHouseCluster
+
+@pytest.fixture(scope="module")
+def started_cluster(request):
+    try:
+        subprocess.check_call(["sysctl", "-w", "net.ipv6.conf.all.disable_ipv6=1"])
+    except subprocess.CalledProcessError:
+        pytest.xfail("IPv6 cannot be modified in this environment")
+
+    cluster = ClickHouseCluster(request.module.__file__)
+    cluster.add_instance("node1", with_zookeeper=True)
```

Comment:
If anything happens here that throws, we're never enabling IPv6 again. We should make a more resilient solution to ensure if we disable IPv6, it's always enabled back after the tests, no matter what.